docs/DEVELOPING.md

```diff
@@ -737,6 +737,140 @@ $ make ftest NAME=mssql
 
 ℹ️ The e2e test uses default values defined in [Configure Microsoft SQL connector](#configure-microsoft-sql-connector)
 
+
+## SharePoint Connector
+
+The [Elastic SharePoint connector](../connectors/sources/sharepoint.py) is provided in the Elastic connectors python framework and can be used via [build a connector](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
+
+### Availability and prerequisites
+
+⚠️ _Currently, this connector is available in **beta** starting in 8.8_.
+Features in beta are subject to change and are not covered by the service level agreement (SLA) of features that have reached general availability (GA).
+
+Elastic versions 8.6.0+ are compatible with Elastic connector frameworks. Your deployment must include the Elasticsearch, Kibana, and Enterprise Search services.
+
```

Comment:
We also need to make it clear what versions of SharePoint the connector is compatible with.

Reply:
+1 

I don't think SharePoint Online has versioning?  It will be helpful to clarify if this is a connector only for Online, or if it can be used for SharePoint Server/Office365.

Additionally, it will be helpful to provide some information on what object types/entities are supported by the SharePoint connector.

Reply:
@leemthompo @ppf2 Let me know if I can resolve this conversation (there sharepoint server versions are listed bellow)